packages/shared/abis/Coordinator.json

```diff
@@ -1,1375 +1,1423 @@
 [
```

Comment:
Opened https://github.com/nucypher/nucypher-ts/issues/310 to use our NPM package instead of manually tracking here. The NPM package probably needs updating, though (cc @manumonti)